pallets/stateful-storage/src/tests.rs

```diff
@@ -2640,3 +2641,22 @@ fn signature_replay_on_deleted_page_check() {
 		));
 	})
 }
+
+#[test]
+fn test_sum_add_actions_bytes() {
```

Comment:
I think you'll need to rebase from main since your test files structure is changed

Reply:
Sounds good. I will rebase the capacity-development branch and update this.